### PR TITLE
0228 add influx line protocol desc

### PR DESCRIPTION
**Pull request by zmstone**
Fixes [EMQX-9036](https://emqx.atlassian.net/browse/EMQX-9036)

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [x] Added tests for the changes
- [x] Changed lines covered in coverage report
- [x] Change log has been added to `changes/{ce,ee}/(feat|perf|fix)-<PR-id>.en.md` and `.zh.md` files
- [x] For internal contributor: there is a jira ticket to track this change
- [~] If there should be document changes, a PR to emqx-docs.git is sent, or a jira ticket is created to follow up
- [~] Schema changes are backward compatible

### CODE REVIEW

File: lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_influxdb.conf [hunk 1]
```diff
@@ -22,14 +22,16 @@ See also [InfluxDB 2.3 Line Protocol](https://docs.influxdata.com/influxdb/v2.3/
 TLDR:</br>
 ```
 <measurement>[,<tag_key>=<tag_value>[,<tag_key>=<tag_value>]] <field_key>=<field_value>[,<field_key>=<field_value>] [<timestamp>]
-```"""
+```
+Please note that a placeholder for an integer value must be annotated with a `i` suffix. For example `${payload.int_value}i`"""
```
Comment (thalesmg) [inline lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_influxdb.conf:26]:
```suggestion
Please note that a placeholder for an integer value must be annotated with an `i` suffix. For example `${payload.int_value}i`"""
```

File: scripts/test/influx/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Test influxdb integration
+
+This script starts two EMQX nodes and a influxdb server in docker container.
+The bootstramping rule engine and data bridge config is provided in influx-bridge.conf
+which got 'includ'ed in the bootstramping config bundle emqx.conf.
```
Comment (thalesmg) [inline scripts/test/influx/README.md:5]:
```suggestion
The bootstrapping rule engine and data bridge config is provided in influx-bridge.conf
which got included in the bootstrapping config bundle emqx.conf.
```

**Review by thalesmg — DISMISSED** [review]

**Review by thalesmg — APPROVED** [review]

**Review by thalesmg — APPROVED** [review]

**Comment by zmstone** [comment]
emqx_pool_SUITE is flaky
